CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
+
```

Comment:
These look dangerous to an unknowing reader. What about a top bullet point

```
### Breaking changes

- Removed core components because they are moved to the cluster-aws app. Make sure to use at least cluster-aws release X.Y.Z when upgrading default-apps-aws to A.B.C or newer.

  - Removed `coredns`
  - Removed [...]
```

Reply:
I left `x.y.z` intentionally. I will update it once we have a cluster-aws release.